modules/encrypt_seclib/src/Tests/EncryptPortability.php

```diff
@@ -0,0 +1,78 @@
+<?php
+namespace Drupal\encrypt\Tests;
+
+/**
+ * Test to make sure an encryption array carries its encryption method and key provider with it to ensure portability.
```

Comment:
Not sure, I just copied it over from encrypt until we revisit testing. I think this was pre-existing from the D7 encrypt